benches/bench.rs

```diff
@@ -12,18 +12,19 @@ mod transform;
 mod me;
 
 use rav1e::*;
-use rav1e::cdef::cdef_filter_frame;
-use rav1e::context::*;
-use rav1e::partition::*;
-use rav1e::predict::*;
-use rav1e::rdo::rdo_cfl_alpha;
-use rav1e::rdo::RDOType;
-use rav1e::ec::*;
+use rav1e::bench::cdef::*;
+use rav1e::bench::context::*;
+use rav1e::bench::ec::*;
+use rav1e::bench::partition::*;
+use rav1e::bench::predict::*;
+use rav1e::bench::rdo::*;
+
 use crate::transform::transform;
 
 use criterion::*;
 use std::time::Duration;
 
+
```

Comment:
nit: extra linebreak

Reply:
I'll remove it now. Thank you for spotting.